src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+
+	export let dApp: DApp;
+	export let backgroundImage: string;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	<div class="absolute">
+		<Img src={backgroundImage} alt={dApp.name} />
+	</div>
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<div class="h-12 w-12 rounded-full">
+				<Img src={dApp.logo} alt={dApp.name} />
+			</div>
+			<div class="flex-1">
+				<h6 class="text-sm font-bold">{$i18n.dapps.text.featured}</h6>
+				<h4 class="text-white">{dApp.name}</h4>
+			</div>
+
+			<Button styleClass="flex-grow-0" colorStyle="secondary" on:click>
```

Comment:
This will opens the modal you mentioned right? Should we maybe add a TODO in some sort?

Reply:
I can, although I'm implementing this right now xD

Reply:
On second thought, no I don't think this is necessary. This component does actually not care what the caller binds to onclick

Reply:
Can you comment the all button for now before merge please and add a TODO?

Reply:
done